examples/apigee/v1beta1/syncauthorization.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: apigee.gcp.upbound.io/v1beta2
+kind: Organization
+metadata:
+  annotations:
+    meta.upbound.io/example-id: apigee/v1beta1/syncauthorization
+    upjet.upbound.io/manual-intervention: "Organization requires a real Project ID"
+  labels:
+    testing.upbound.io/example-name: apigee-sync-authorization
+  name: apigee-sync-authorization
+spec:
+  forProvider:
+    analyticsRegion: us-central1
+    runtimeType: HYBRID
+    projectId: ${project.project_id}
+
+---
+
+apiVersion: apigee.gcp.upbound.io/v1beta1
+kind: SyncAuthorization
+metadata:
+  annotations:
+    meta.upbound.io/example-id: apigee/v1beta1/syncauthorization
+    upjet.upbound.io/manual-intervention: "This resource was skipped because the dependent Organization resource was skipped"
+  labels:
+    testing.upbound.io/example-name: apigee-sync-authorization
+  name: ${project.project_id}
```

Comment:
My only doubt is that since we moved the name field to metadata due to the external name configuration, we removed the possibility of using selectors and references for this field.

Reply:
Good point let me think about

Reply:
in case we set the name via spec.forProvider.name what we need to configure for 

`"google_apigee_sync_authorization": config.TemplatedStringAsIdentifier("name", "organizations/{{ .external_name }}/syncAuthorization"),` - `IdentifiedByProvider` ?

Reply:
We can try the following configuration:

```
"google_apigee_sync_authorization": config.TemplatedStringAsIdentifier("", "organizations/{{ .parameters.name }}/syncAuthorization")
```
With the above configuration, the `name` field will be moved to `spec.forProvider`, but we will have a more complex external name.

On second thought, we can also handle the selection with compositions. So, I am not opinionated on how we proceed here. I leave the decision to you.